lib/check_search.rb

```diff
@@ -488,8 +488,8 @@ def file_filter
 
   def team_tasks_conditions
     conditions = []
-    return conditions unless @options.has_key?('team_tasks') && @options['team_tasks'].class.name == 'Array'
-    @options['team_tasks'].delete_if{ |tt| tt['response'].blank? }
+    return conditions unless @options['team_tasks'].class.name == 'Array'
```

Comment:
why did you remove `@options.has_key?('team_tasks')` from unless condition?
I think we should keep it and only update `delete_if` to include `tt['id'].blank?`

Reply:
@melsawy because I think it's redundant with `@options['team_tasks'].class.name` ... is there is no `team_tasks` key, `@options['team_tasks'].class.name` is going to return `NilClass`